src/apprunner/wizards/codeRepositoryWizard.ts

```diff
@@ -42,10 +42,11 @@ function validateCommand(command: string): string | undefined {
 }
 
 function createRepoPrompter(git: GitExtension): QuickPickPrompter<Remote> {
-    const remotes = git.getRemotes()
+    const mapRemote = (remote: Remote) => ({ label: remote.name, detail: remote.fetchUrl, data: remote })
+    const remotes = git.getRemotes().then(r => r.map(mapRemote))
```

Comment:
Why use a `.then` instead of an `await`? Assuming it's not too big of an issue since I'm assuming(?) VS Code auto-activates the Git extension, but this could lead to issues if `remotes` is not assigned by the time the qp is created.

Do we really gain anything from `mapRemote` being specified as a separate const?

Reply:
I think it's to drive the point that this is asynchronous. 

> Do we really gain anything from mapRemote being specified as a separate const?

Hmm yeah looks verbose.